pkgs/development/python-modules/python-yate/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, aiohttp
+, async-timeout
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "python-yate";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "eventphone";
+    repo = "python-yate";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-AdnlNsEOFuzuGTBmfV9zKyv2iFHEJ4eLMrC6SHHf7m0=";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
```

Comment:
`aiohttp` does not seem to be a required depencency? https://github.com/eventphone/python-yate/blob/e58fb6f083386e9089e977f404f2e466c3e6cd4f/setup.py#L29

Reply:
Looks like that upstream [missed to mention](https://github.com/eventphone/python-yate/blob/master/yate/callgen.py#L7) it.

Reply:
Yes, this is true